definitions/contested/json/CaseField/CaseField.json

```diff
@@ -2677,6 +2677,14 @@
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "12/05/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "consentOrderFRCText",
+    "Label": "Court Details",
+    "FieldType": "Text",
```

Comment:
It may also be worth having the field as a Label, not sure it really matters

Reply:
I tried doing that, I was getting CCD validation error, something about not having validation for label